pycbc/inference/sampler/emcee_pt.py

```diff
@@ -107,20 +115,48 @@ def betas(self):
 
     @classmethod
     def from_config(cls, cp, model, nprocesses=1, use_mpi=False):
-        """Loads the sampler from the given config file."""
+        """
+        Loads the sampler from the given config file.
+
+        For generating the temperature ladder to be used by emcee_pt, either
+        the number of temperatures (provided under a subsection 'ntemps'),
+        or the path to a file storing inverse temperature values (provided 
+        under a subsection inverse-temperatures-file) can be loaded from the
```

Comment:
Change "provided under a subsection" to "provided by the option" here and at line 122.